docs/src/main/sphinx/connector/oracle.rst

```diff
@@ -298,9 +298,6 @@ For Oracle ``NUMBER`` (without precision and scale), you can change
 Mapping datetime types
 ^^^^^^^^^^^^^^^^^^^^^^
 
-Selecting a timestamp with fractional second precision (``p``) greater than 3
```

Comment:
Now for precision greater than `9` will still be truncated right ? If we are throwing an exception for higher precision, then for some queries might be backward incompatible right ?

Reply:
I added such test: 
```
 .addRoundTrip("TIMESTAMP '1970-01-01 00:00:00.987987987111'", "TIMESTAMP '1970-01-01 00:00:00.987987987'")
                .addRoundTrip("TIMESTAMP '1970-01-01 00:00:00.987987987999'", "TIMESTAMP '1970-01-01 00:00:00.987987988'")

        tests.execute(getQueryRunner(), session, trinoCreateAsSelect(session, "test_timestamp"));
        tests.execute(getQueryRunner(), session, trinoCreateAsSelect("test_timestamp"));
        tests.execute(getQueryRunner(), session, trinoCreateAndInsert(session, "test_timestamp"));
        tests.execute(getQueryRunner(), session, trinoCreateAndInsert("test_timestamp"));
```

So as far as I understand now for precisions greater then 9, we round

Reply:
If we create a table like this 
```
create table tes (t timestamp(12));
```

Show create corresponds to 
```
 CREATE TABLE oracle.trino_test.tes (
    t timestamp(9)
 )
```

Should we mention them in the docs.